### PR TITLE
timerdevice: fix error to get hardware time

### PR DESCRIPTION
**Pull request by sitoliu**
ID: 1544352
Signed-off-by: Sitong Liu <siliu@redhat.com>

### CODE REVIEW

**Comment by sitoliu** [comment]
Depends on: https://github.com/avocado-framework/avocado-vt/pull/1350

File: qemu/tests/cfg/timedrift_adjust_time.cfg [hunk 1]
```diff
@@ -1,5 +1,5 @@
 - timedrift_adjust_time:
-    no RHEL.3,4,5 WinXp Win2000 Win2003 WinVista
+    no RHEL.3, RHEL.4, RHEL.5, WinXp, Win2000, Win2003, WinVista
```
Comment (hereischen) [inline qemu/tests/cfg/timedrift_adjust_time.cfg]: Don't need to add `no RHEL.3, RHEL.4, RHEL.5`, avocado it self does no support them anymore.

**Comment by hereischen** [comment]
Please provide a detailed commit log.
eg, what are the errors? why this solution?

File: qemu/tests/cfg/timedrift_adjust_time.cfg (hunk 1 above)
Comment (sitoliu) [reply qemu/tests/cfg/timedrift_adjust_time.cfg]: Thanks, will update it.

**Comment by sitoliu** [comment]
> Please provide a detailed commit log.
eg, what are the errors? why this solution?

When try to get expected format of hardware time, add `LC_TIME=C` before command `hwclock -u`, and this operation is already in `utils_test.get_time()`, so remove it.

**Comment by sitoliu** [comment]
JOB LOG    : /root/avocado/job-results/job-2018-06-25T22.48-e9b1a36/job.log
 (1/1) Host_RHEL.m7.u6.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.x86_64.io-github-autotest-qemu.timedrift_no_net.guest_pause_resume.clock_vm: PASS (138.55 s)

**Comment by sitoliu** [comment]
/root/avocado/job-results/job-2018-06-26T11.55-7a13959/job.log
 (1/1) Host_RHEL.m7.u6.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.timedrift_adjust_time.guest_reboot.clock_vm.adjust_host_clock: PASS (328.84 s)

**Comment by hereischen** [comment]
Acked-by: Haotong Chen hachen@redhat.com

File: qemu/tests/timedrift_no_net.py [hunk 2]
```diff
@@ -148,7 +148,7 @@ def run(test, params, env):
     session.close()
 
     if sub_work in globals():  # Try to find sub work function.
-        globals()[sub_work](params, vm, session)
+        globals()[sub_work](test, params, vm, session)
```
Comment (luckyh) [inline qemu/tests/timedrift_no_net.py]: Should have the same change for `timedrift_no_net_win.py`, right?

Comment (sitoliu) [reply qemu/tests/timedrift_no_net.py]: Thanks, modified it.

**Pull request merged**